otter/cloud_client.py

```diff
@@ -649,7 +652,7 @@ def _process_clb_api_error(api_error_code, json_body, lb_id):
           partial(CLBRateLimitError, lb_id=lb_id))] +
         _expand_clb_matches(
             [(422, _CLB_DELETED_PATTERN, CLBDeletedError),
-             (422, _CLB_PENDING_UPDATE_PATTERN, CLBPendingUpdateError),
+             (422, _CLB_IMMUTABLE_PATTERN, CLBImmutableError),
```

Comment:
Non-blocking - wondering if we should comment here that this particular ordering matters, otherwise a PENDING_DELETE server will be recognized as just a `CLBImmutableError`?